packages/preview/treet/0.1.0/typst.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "treet"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["8LWXpg"]
+license = "MIT"
+description = "make tree list easily"

```

```suggestion
description = "Create tree lists easily"
```
Just a suggestion :)